[CI] SplitIndexIT testCreateSplitIndexToN failing

**Build scan:**
https://gradle-enterprise.elastic.co/s/bvb5puxxlbaou/tests/:server:internalClusterTest/org.elasticsearch.action.admin.indices.create.SplitIndexIT/testCreateSplitIndexToN

**Reproduction line:**
```
gradlew ':server:internalClusterTest' --tests "org.elasticsearch.action.admin.indices.create.SplitIndexIT.testCreateSplitIndexToN" -Dtests.seed=24641F2A0356AEBD -Dtests.locale=ar-MA -Dtests.timezone=SystemV/EST5 -Druntime.java=19
```

**Applicable branches:**
8.6

**Reproduces locally?:**
Didn't try

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.action.admin.indices.create.SplitIndexIT&tests.test=testCreateSplitIndexToN

**Failure excerpt:**
```
java.lang.AssertionError: timed out waiting for green state

  at __randomizedtesting.SeedInfo.seed([24641F2A0356AEBD:92C8F9F3E9C1145A]:0)
  at org.junit.Assert.fail(Assert.java:88)
  at org.elasticsearch.test.ESIntegTestCase.ensureColor(ESIntegTestCase.java:966)
  at org.elasticsearch.test.ESIntegTestCase.ensureGreen(ESIntegTestCase.java:905)
  at org.elasticsearch.test.ESIntegTestCase.ensureGreen(ESIntegTestCase.java:894)
  at org.elasticsearch.action.admin.indices.create.SplitIndexIT.splitToN(SplitIndexIT.java:245)
  at org.elasticsearch.action.admin.indices.create.SplitIndexIT.testCreateSplitIndexToN(SplitIndexIT.java:78)
  at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
  at java.lang.reflect.Method.invoke(Method.java:578)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:44)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
  at java.lang.Thread.run(Thread.java:1589)

```

` [2022-12-07T03:09:51,640][INFO ][o.e.a.a.i.c.SplitIndexIT ] [testCreateSplitIndexToN] ensureGreen timed out, cluster state:`

The test timed out and it seems to me that these exceptions are relevant:

```
  1> [2022-12-07T03:09:14,738][WARN ][o.e.i.IndicesService     ] [node_s2] [target/zDozxK_RTPCd927QhZfX2g] failed to delete index
  1> java.io.IOException: could not remove the following files (in the order of attempts):
  1>    C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\1\index\_2.cfs: java.nio.file.AccessDeniedException: C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\1\index\_2.cfs
  1>    C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\1\index\_1.cfs: java.nio.file.AccessDeniedException: C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\1\index\_1.cfs
  1>    C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\1\index\_0.cfs: java.nio.file.AccessDeniedException: C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\1\index\_0.cfs
  1>    C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\1\index: java.nio.file.DirectoryNotEmptyException: C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\1\index
  1>    C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\1: java.nio.file.DirectoryNotEmptyException: C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\1
  1>    C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\0\index\_2.cfs: java.nio.file.AccessDeniedException: C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\0\index\_2.cfs
  1>    C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\0\index\_1.cfs: java.nio.file.AccessDeniedException: C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\0\index\_1.cfs
  1>    C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\0\index\_0.cfs: java.nio.file.AccessDeniedException: C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\0\index\_0.cfs
  1>    C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\0\index: java.nio.file.DirectoryNotEmptyException: C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\0\index
  1>    C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\0: java.nio.file.DirectoryNotEmptyException: C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g\0
  1>    C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g: java.nio.file.DirectoryNotEmptyException: C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s2\indices\zDozxK_RTPCd927QhZfX2g
```

```
1> Caused by: java.nio.file.AccessDeniedException: C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s0\indices\l7teugWcQ_GXYt5PKOYV_w\7\index\recovery.Lfot0VCSR16-vueYBDdx1A._0.cfs -> C:\Users\jenkins\workspace\platform-support\42\server\build\testrun\internalClusterTest\temp\org.elasticsearch.action.admin.indices.create.SplitIndexIT_24641F2A0356AEBD-001\tempDir-002\node_s0\indices\l7teugWcQ_GXYt5PKOYV_w\7\index\_0.cfs
  1> 	at sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:89)
  1> 	at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
  1> 	at sun.nio.fs.WindowsFileCopy.move(WindowsFileCopy.java:317)
  1> 	at sun.nio.fs.WindowsFileSystemProvider.move(WindowsFileSystemProvider.java:293)
  1> 	at org.apache.lucene.tests.mockfile.FilterFileSystemProvider.move(FilterFileSystemProvider.java:144)
  1> 	at org.apache.lucene.tests.mockfile.FilterFileSystemProvider.move(FilterFileSystemProvider.java:144)
  1> 	at org.apache.lucene.tests.mockfile.FilterFileSystemProvider.move(FilterFileSystemProvider.java:144)
  1> 	at org.apache.lucene.tests.mockfile.FilterFileSystemProvider.move(FilterFileSystemProvider.java:144)
  1> 	at java.nio.file.Files.move(Files.java:1430)
  1> 	at org.apache.lucene.store.FSDirectory.rename(FSDirectory.java:272)
  1> 	at org.apache.lucene.store.FilterDirectory.rename(FilterDirectory.java:91)
  1> 	at org.apache.lucene.store.FilterDirectory.rename(FilterDirectory.java:91)
  1> 	at org.elasticsearch.index.store.Store.renameTempFilesSafe(Store.java:325)
  1> 	at org.elasticsearch.indices.recovery.MultiFileWriter.renameAllTempFiles(MultiFileWriter.java:223)
  1> 	at org.elasticsearch.indices.recovery.RecoveryTarget.lambda$cleanFiles$6(RecoveryTarget.java:494)
  1> 	at org.elasticsearch.action.ActionListener.completeWith(ActionListener.java:444)
  1> 	... 10 more
```

For the record, the failures seem to have been mostly in the past 3/4 months, and on Windows, and the few I've checked all seem to have had the `Caused by: java.nio.file.AccessDeniedException` in their logs. Not sure why this wasn't happening before. 
<img width="1177" alt="image" src="https://user-images.githubusercontent.com/3340727/216984291-f9a83230-f5a2-45b4-8787-01a721577d4a.png">

--- a/server/src/internalClusterTest/java/org/elasticsearch/action/admin/indices/create/SplitIndexIT.java
+++ b/server/src/internalClusterTest/java/org/elasticsearch/action/admin/indices/create/SplitIndexIT.java
@@ -72,15 +72,15 @@ public class SplitIndexIT extends ESIntegTestCase {
         return false;
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/92183")
     public void testCreateSplitIndexToN() throws IOException {
+        assumeFalse("https://github.com/elastic/elasticsearch/issues/34080", Constants.WINDOWS);
+
         int[][] possibleShardSplits = new int[][] { { 2, 4, 8 }, { 3, 6, 12 }, { 1, 2, 4 } };
         int[] shardSplits = randomFrom(possibleShardSplits);
         splitToN(shardSplits[0], shardSplits[1], shardSplits[2]);
     }
 
     public void testSplitFromOneToN() {
-
         assumeFalse("https://github.com/elastic/elasticsearch/issues/34080", Constants.WINDOWS);
 
         splitToN(1, 5, 10);

--- a/server/src/internalClusterTest/java/org/elasticsearch/action/admin/indices/create/SplitIndexIT.java
+++ b/server/src/internalClusterTest/java/org/elasticsearch/action/admin/indices/create/SplitIndexIT.java
@@ -73,7 +73,7 @@ public class SplitIndexIT extends ESIntegTestCase {
     }
 
     public void testCreateSplitIndexToN() throws IOException {
-        assumeFalse("https://github.com/elastic/elasticsearch/issues/34080", Constants.WINDOWS);
+        assumeFalse("https://github.com/elastic/elasticsearch/issues/33857", Constants.WINDOWS);
 
         int[][] possibleShardSplits = new int[][] { { 2, 4, 8 }, { 3, 6, 12 }, { 1, 2, 4 } };
         int[] shardSplits = randomFrom(possibleShardSplits);

--- a/server/src/internalClusterTest/java/org/elasticsearch/action/admin/indices/create/SplitIndexIT.java
+++ b/server/src/internalClusterTest/java/org/elasticsearch/action/admin/indices/create/SplitIndexIT.java
@@ -81,7 +81,7 @@ public class SplitIndexIT extends ESIntegTestCase {
     }
 
     public void testSplitFromOneToN() {
-        assumeFalse("https://github.com/elastic/elasticsearch/issues/34080", Constants.WINDOWS);
+        assumeFalse("https://github.com/elastic/elasticsearch/issues/33857", Constants.WINDOWS);
 
         splitToN(1, 5, 10);
         client().admin().indices().prepareDelete("*").get();

